app/managers/security_manager/index.test.ts

```diff
@@ -394,14 +394,25 @@ describe('SecurityManager', () => {
     });
 
     describe('goToPreviousServer', () => {
+        afterAll(() => {
```

Comment:
if we have this in the beforeEach of the parent block, why do we need this?

https://github.com/mattermost/mattermost-mobile/blob/cefc4cb32019d5aae4e24cdcd26bfdf2103ec443/app/managers/security_manager/index.test.ts#L42

Reply:
Because I missed it hahahha.. will fix in an upcoming unrelated PR :)